src/naemon/checks_service.c

```diff
@@ -466,7 +466,7 @@ int handle_async_service_check_result(service *temp_service, check_result *queue
 	else {
 
 		/* parse check output to get: (1) short output, (2) long output, (3) perf data */
-		parse_check_output(queued_check_result->output, &temp_service->plugin_output, &temp_service->long_plugin_output, &temp_service->perf_data, TRUE, FALSE);
+		parse_check_output(queued_check_result->output, &temp_service->plugin_output, &temp_service->long_plugin_output, &temp_service->perf_data, TRUE);
```

Comment:
Since there might exist NEB-modules that call this function, perhaps we should bump CURRENT_NEB_API_VERSION?

Reply:
or should we use __attribute__ ((unused)) for now?

Reply:
I think that sounds like a good idea!

Reply:
Did you check to see if these were the only occurrences of that warning? I suspect that, as we compile with "-Wno-unused-parameter", there could be several places this occurs.

I am fine with using the __attribute__ annotation.